src/Hedgehog.Experimental/Gen.fs

```diff
@@ -228,17 +229,29 @@ module GenX =
           String = Gen.string (Range.linear 0 50) Gen.latin1
           DateTime = Gen.dateTime
           DateTimeOffset = Gen.dateTime |> Gen.map System.DateTimeOffset
-          SeqRange = Range.exponential 0 50 }
+          SeqRange = Range.exponential 0 50
+          RecursionDepth = 1 }
+
+    let rec private auto''<'a> (config : AutoGenConfig) (recursionDepths: Map<string, int>) : Gen<'a> =
```

Comment:
Having *two* primes on a name, hmm...

Perhaps `autoWith`?

At this point I can tell that you prefer short names, but I think `auto''` might be confusing(?)

Reply:
It's a private function so I don't think it matters that much. But I like `autoWith`, I suggest we call the private function `autoInner`, and rename `auto'` (the public function that takes a config) to `autoWith`.

Reply:
:+1: